6.5/cdm/README.md

```diff
@@ -22,13 +22,20 @@ to such license between the licensee and ForgeRock AS.
 - Single realm
 - OAuth 2.0 Server and Client
 - Separate userstore, configstore and ctsstore
+- CTS with Affinity across two servers
 - LDAP Auth module default
-- To log into admin console use https://fqdn/XUI/?service=adminconsoleservice
+- Stateful sessions and stateful OAuth 2.0
+- All connection pools tuned
+- Using DS TTL for token reaping and the new GrantSet Token Storage Scheme
+	- Refer to the product documents on some limitions on notifications caused by using DS TTL
+- To log into admin console specify the "adminconsoleservice" as query parameter.  For example
+	- https://login.example.com/XUI/?service=adminconsoleservice
```

Comment:
Would be nice to have the namespace in this example, i.e., login.prod.example.com